source/_docs/agency-tips.md

```diff
@@ -75,10 +75,6 @@ New Relic APM Pro is an advanced tool for application monitoring and troubleshoo
 ## Cron
 Pantheon does not provide a way to set custom scheduling for cron jobs. For details, see [Cron for Drupal](/docs/drupal-cron) and [Cron for WordPress](/docs/wordpress-cron).
 
-## Kalabox
```

Comment:
Can we convert this section to a Lando section, rather than removing a recommendation for local development?

Reply:
@rachelwhitton I didn't convert this to a Lando section because I was worried about providing an "official" recommendation while Lando is still in its infancy, and not necessarily ready for prime time, as it were.

@dustinleblanc what are your thoughts on this?

Reply:
This is a very very tricky subject. As a maitainer for Lando, half of me is always very happy to recommend the project to people, and I *do* think it's the best solution for local dev with Pantheon. However, with my CSE hat firmly in place, and coming off this morning's conversation about support scope, suggesting NO specific local dev environment is easiest for CSE to support. Lando while great, is definitely still buggy, particularly this next release is likely to have lots of bugs due to our refactoring. This will likely soon be a moot point for reasons we are all aware, so the solution needed here is a stop gap. Given that, the current state of Lando, and my sort of 'personal conflict of interest', I'd say the wisest move is to stay silent on the subject and only recommend lando to folks in a one on one, non-official support offering kind of way. Power Users is a great place for this, I myself need to get better at not talking to widely about it in official support channels.

Reply:
Makes sense - let's ignore my suggestion here